clang/test/CheckedCRewriter/checkedregionvoid.c

```diff
@@ -0,0 +1,9 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+#include "dummy.h"
+// Dummy to cause output
+void f(int *x) {}
```

Comment:
Add lines to your test to ensure they are *not* rewritten. I.e., something like `// CHECK: int f(int *x) {}` here.